components/HomePage/WhyIronFish/WhyIronFish.tsx

```diff
@@ -1,32 +1,77 @@
 import {
   Box,
+  Grid,
   Container,
   Text,
   StickySideBySideView,
   ShadowBox,
+  Flex,
+  FancyArrowRight,
 } from "@/lib/ui";
 import { ReactNode } from "react";
 import { Decentralized } from "./icons/Decentralized";
 import { Handshake } from "./icons/Handdhake";
 import { Lock } from "./icons/Lock";
+import Link from "next/link";
 
 const sections = [
   {
     heading: "Privacy without compromise",
     description:
-      "Iron Fish is built from the ground up with zk-SNARK technology. Integrating encryption solutions at the foundation of our protocol delivers maximal security across the platform.",
+      "Every crypto asset deserves access to a privacy layer. With Iron Fish you can transfer any asset in a safe and private way.",
     image: <Lock />,
   },
   {
     heading: "Your finance, truly decentralized",
     description:
-      "Powered by a global network of miners and nodes, Iron Fish is a censorship resistant, resilient, and privacy-enabled platform for everyday crypto transactions.",
+      "Powered by a global network of miners and nodes, Iron Fish is censorship resistant and offers a privacy-enabled platform for everyday crypto transactions.",
     image: <Decentralized />,
   },
   {
     heading: "A common-sense approach to compliance",
-    description:
-      "With an Iron Fish account, users receive a set of view keys that allow them to provide read-only access to their transactions records. This feature enables users to share account information for compliance and situations requiring proof of transaction.",
+    description: (
+      <Grid gap={4}>
+        <Text>
+          Every transfer in and out of Iron Fish passes through ChainPort&apos;s
+          sanction screening and real-time threat detection models. We take a
+          strong stance that Iron Fish is to be used for good, and malicious
+          addresses attempting to bridge to Iron Fish will have their funds
+          frozen.
+        </Text>
+        <Flex mt="">
```

Comment:
super small nit: we can get rid of the `mt`

Reply:
And probably this `<Flex>` all together if we add a `w="fit-content"` to the child